app/scripts/views/choose_what_to_sync.js

```diff
@@ -121,6 +123,8 @@ define(function (require, exports, module) {
         });
       }
     }
+  }, {
+    SCREEN_CLASS
```

Comment:
I exposed it for the tests, to ensure the class was added/removed. Sorry for the orthogonal change, while adding the test for destroy, I noticed the addition/removal of the class name was not tested for either.